ember-cli-build.js

```diff
@@ -11,9 +9,8 @@ module.exports = function (defaults) {
 
   const app = new EmberApp(defaults, {
     fingerprint: {
-      extensions: broccoliAssetRevDefaults.extensions.concat(['webmanifest', 'svg']),
```

Comment:
Was this removed on purpose? I'm not certain it's needed here, but I'm not sure it isn't either.